src/index.js

```diff
@@ -11,8 +13,14 @@ import {
   exec,
   normalizeSourceMap,
   normalizeSourceMapAfterPostcss,
+  parsePackageJson,
+  isFileExists,
 } from "./utils";
 
+let hasExplicitDependencyOnPostCSS = false;
+
+const PACKAGE_JSON_PATH = path.resolve(process.cwd(), "package.json");
```

Comment:
cwd can be `src` (i.e. `src/package.json`), but it is invalid I provided example above with recusing searching

Reply:
it would be possible to use iterators to search for a file (from similar solutions), but it seems that this is not essential here. I took your solution, especially since there is a similar solution in the Webpack source code.

Reply:
I also use `null and `undefined` to cache calls. For some, this may seem implicit. I can try to make it more explicit, if necessary.